plugins/inputs/jenkins/jenkins.go

```diff
@@ -193,6 +193,10 @@ func (j *Jenkins) gatherNodeData(n node, acc telegraf.Accumulator) error {
 	tags["disk_path"] = monitorData.HudsonNodeMonitorsDiskSpaceMonitor.Path
 	tags["temp_path"] = monitorData.HudsonNodeMonitorsTemporarySpaceMonitor.Path
 
+	urlTokens := strings.Split(j.URL, ":")
+	tags["source"] = urlTokens[1][2:]
+	tags["port"] = urlTokens[2]
```

Comment:
Before doing any indexing make sure the expected number of items are in the array, otherwise this could panic.  However, for URL parsing, its best to just use the `url.Parse` function.

Reply:
I'll switch this over.  I saw `net.SplitHostPort()` which didn't like this url format  but not `url.parse`.